src/js/pages/settings/UISettingsPage.js

```diff
@@ -0,0 +1,105 @@
+import React from "react";
+import { Trans } from "@lingui/macro";
+import { i18nMark } from "@lingui/react";
+import { Link } from "react-router";
+
+import Breadcrumb from "#SRC/js/components/Breadcrumb";
+import BreadcrumbTextContent from "#SRC/js/components/BreadcrumbTextContent";
+import ConfigurationMap from "#SRC/js/components/ConfigurationMap";
+import ConfigurationMapHeading from "#SRC/js/components/ConfigurationMapHeading";
+import ConfigurationMapLabel from "#SRC/js/components/ConfigurationMapLabel";
+import ConfigurationMapRow from "#SRC/js/components/ConfigurationMapRow";
+import ConfigurationMapSection from "#SRC/js/components/ConfigurationMapSection";
+import ConfigurationMapValue from "#SRC/js/components/ConfigurationMapValue";
+import EventTypes from "#SRC/js/constants/EventTypes";
+import Languages from "#SRC/js/constants/Languages";
+import Loader from "#SRC/js/components/Loader";
+import Page from "#SRC/js/components/Page";
+import LanguageModalStore from "#SRC/js/stores/LanguageModalStore";
+import LanguageModalActions from "#SRC/js/events/LanguageModalActions";
+import UserLanguageStore from "#SRC/js/stores/UserLanguageStore";
+
+const Loading = () => <Loader size="small" type="ballBeat" />;
+
+const UISettingsBreadcrumbs = () => {
+  const crumbs = [
+    <Breadcrumb key={-1} title={i18nMark("UI Settings")}>
```

Comment:
Will need to be pretranslated using `withI18n` and `this.props.i18n._(t`UI Settings`)`